docs/docs/platform/activity-feed.md

```diff
@@ -7,7 +7,8 @@ sidebar_position: 1
 Novu provides a notification activity feed that monitors every outgoing message associated with its relevant metadata.
 This can be used to monitor activity and discover potential issues with a specific provider or a channel type.
 
-[Activity Screen](/img/activity.png)
```

Comment:
@BoseSj 
This issue will be fixed by adding ! before [Activity Screen]
Right now it is behaving like a link

Reply:
Okay 😄

Reply:
adding `!` before the current line not quite solving the issue in markdown
the output >>
![image](https://user-images.githubusercontent.com/58129377/194747573-b8fe81e7-3ff6-422a-aa2f-00252dd0c21c.png)

Reply:
I think the solution is just to replace the link with `https://user-images.githubusercontent.com/../` one as it s done in [architecture.md](https://github.com/novuhq/novu/blob/87c95ead141c117bef75fd5fdf75752426cb0cbe/docs/docs/overview/architecture.md) file. (Just copy-pasting the image.) In that case its working properly both in markdown and the site.